bb/object_pool.c

```diff
@@ -312,6 +310,7 @@ int comdb2_objpool_destroy(comdb2_objpool_t op)
 
     /* clear access history */
     hash_for(op->history, hash_elem_free_wrapper, NULL);
+    hash_clear(op->history);
```

Comment:
You don't need to call `hash_clear()` as `hash_free()` clears entries as well.

Reply:
good point, removing, thanks.